src/schema/objects/metadata.yaml

```diff
@@ -2702,6 +2844,22 @@ RepetitionTime:
   type: number
   exclusiveMinimum: 0
   unit: s
+RepetitionTime__mrs:
+  name: RepetitionTime
+  display_name: Repetition Time
+  description: |
+    The time between repetitions (TR), specified in seconds.
+    For dynamic acquisitions, such as cardiac-triggered acquisitions where the TR
+    may differ over the scan, an array of numbers can be used.
```

Comment:
This would correspond to `RepetitionTimeExcitation`, [DICOM Tag 0018, 0080](https://dicom.innolitics.com/ciods/mr-image/mr-image/00180080). But `RepetitionTime` seems to make more sense. I've done it this way to permit arrays of numbers (like in the case of cardiac triggering). But that said, `VolumeTiming` would also technically fit the case of such dynamic TRs.